src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/TensorPrimitives.netcore.cs

```diff
@@ -13715,7 +13715,7 @@ public static Vector512<float> Invoke(Vector512<float> x)
                     // (x < 0) ? float.NaN : x
                     Vector512<float> lessThanZeroMask = Vector512.LessThan(x, Vector512<float>.Zero);
 
-                    specialResult = Vector512.ConditionalSelect(
+                    specialResult = ElementWiseSelect(
```

Comment:
I only changed ConditionalSelect calls that were targeting the result of an Equals/GreaterThan/LessThan/etc. But are there any ConditionalSelect calls in this file that couldn't be ElementWiseSelect instead?

Reply:
It's basically anything where we can guarantee that its `AllBitsSet` or `Zero` on a per-element basis. This is most naturally the comparison operations but can also apply to some bitwise operations of those.

Most other operations don't guarantee such an output, so it'd likely be a case by case basis.